sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -157,152 +148,16 @@ class SRSClient(GenericClient):
     <BLANKLINE>
 
     """
-    BASE_URL = 'ftp://ftp.swpc.noaa.gov/pub/warehouse/'
-    MIN_YEAR = 1996
+    baseurl = r'ftp://ftp.ngdc.noaa.gov/STP/swpc_products/daily_reports/solar_region_summaries/%Y/%m/%Y%m%dSRS.txt'
+    pattern = '{}/{year:4d}/{month:2d}/{year:4d}{month:2d}{day:2d}SRS.txt'
 
-    def _get_url_for_timerange(self, timerange):
```

Comment:
I am not opposed in any way to code removal and I have no doubt this is fine, but why are we removing all of this code? There could maybe be a line in the change log (or just an additional comment on the PR even) about the reasoning behind this refactor.

Reply:
This client was never refactored with the GenericClient refactor from GSoC as it had tar files and therefore needed all this custom code. The new FTP url only has text files and therefore can be made very simple now.